draft-ietf-openpgp-pqc.md

```diff
@@ -972,10 +946,10 @@ In the random oracle setting, the reordering does not influence the arguments in
 
 ## Domain separation and binding {#sec-fixed-info}
 
-The `domSeparation` information defined in {{kem-fixed-info}} provides the domain separation for the key combiner construction.
+The `domSeparation` information defined in {{kem-key-combiner}} provides the domain separation for the key combiner construction.
 This ensures that the input keying material is used to generate a KEK for a specific purpose or context.
 
-The `algID` defined in {{kem-fixed-info}} binds the derived KEK to the chosen algorithm and communication parties.
+The `algID` defined in {{kem-key-combiner}} binds the derived KEK to the chosen algorithm and communication parties.
```

Comment:
I think this wording was more natural when it referred to `fixedInfo`. Now it's weird to refer to the `algID` input like this so we should rephrase.

Reply:
```suggestion
The `algID` defined in {{kem-key-combiner}} binds the derived KEK to the chosen algorithm.
```

Reply:
I had already made the above suggestion, only that I was still "review mode" for some reason and didn't commit it.